src/tracks/playabletrack/wavetrack/ui/SampleHandle.cpp

```diff
@@ -290,9 +300,8 @@ UIHandle::Result SampleHandle::Click
       }
       //Set the sample to the point of the mouse event
```

Comment:
This old comment should instead say, set a range of samples around the mouse event